doc/user_guide/metadata_structure.rst

```diff
@@ -300,6 +348,11 @@ record_by
     One of 'spectrum' or 'image'. It describes how the data is stored in memory.
     If 'spectrum' the spectral data is stored in the faster index.
 
+quantity
+    type: Str
+    
+    The name of the quantity of the "intensity axis" with the units in round brackets if required.
```

Comment:
The name of the quantity of the signal with the units in round brackets if required. For example Temperature (K)?